### PR TITLE
pythonPackages.yubico-client: init at 1.10.0

### PR DESCRIPTION
**Pull request by peterromfeldhk**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/yubico-client/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  disabled = isPyPy;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0skkmrpvpb1pwyqjf3lh9vq46xagvwdx9kagpdbba2v5dgrk34d1";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  doCheck = false;
```
Comment (worldofpeace) [inline pkgs/development/python-modules/yubico-client/default.nix]: We'd like to enable tests. Why have they been disabled?

File: pkgs/development/python-modules/yubico-client/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  meta = with lib; {
+    description = "Python class for verifying Yubico One Time Passwords (OTPs) based on the validation protocol version 2.0";
```
Comment (worldofpeace) [inline pkgs/development/python-modules/yubico-client/default.nix]: Other meta attributes that are good to have are

-  homepage
-  maintainers

> https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

**Review by worldofpeace — CHANGES_REQUESTED** [review]
Hi, thanks for your contribution.

Here's the changes we need to get this merged.

File: pkgs/development/python-modules/yubico-client/default.nix (hunk 1 above)
Comment (peterromfeldhk) [reply pkgs/development/python-modules/yubico-client/default.nix]: i got
```
ModuleNotFoundError: No module named 'test_utils'
```

Comment (peterromfeldhk) [reply pkgs/development/python-modules/yubico-client/default.nix]: i guess i can do anther fetchfromgit

File: pkgs/development/python-modules/yubico-client/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    rev = "${version}";
+    sha256 = "0rqma8iqwkmxrpmdnkrsr6425r9vgjnbw0j7rp1p13xb1alkjs6q";
+  };
+
+  checkInputs = [ python-dateutil pytz ];
+
+  propagatedBuildInputs = [ requests ];
+
+  meta = with lib; {
+    description = "Python class for verifying Yubico One Time Passwords (OTPs) based on the validation protocol version 2.0";
```
Comment (dotlambda) [inline pkgs/development/python-modules/yubico-client/default.nix]: too long

File: pkgs/development/python-modules/yubico-client/default.nix [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    homepage = "https://github.com/Kami/python-yubico-client/";
```
Comment (dotlambda) [inline pkgs/development/python-modules/yubico-client/default.nix]:
```suggestion
    homepage = https://github.com/Kami/python-yubico-client/;
```

File: pkgs/development/python-modules/yubico-client/default.nix [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+buildPythonPackage rec {
+  pname = "yubico-client";
+  version = "1.10.0";
+
+  disabled = isPyPy;
+
+  src = fetchFromGitHub {
+    owner = "Kami";
+    repo = "python-${pname}";
+    rev = "${version}";
```
Comment (dotlambda) [inline pkgs/development/python-modules/yubico-client/default.nix]:
```suggestion
    rev = "version";
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/yubico-client/default.nix [hunk 6]
```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchFromGitHub
+, requests, python-dateutil, pytz }:
+
+buildPythonPackage rec {
+  pname = "yubico-client";
+  version = "1.10.0";
+
+  disabled = isPyPy;
```
Comment (dotlambda) [inline pkgs/development/python-modules/yubico-client/default.nix]: According to https://github.com/Kami/python-yubico-client/blob/master/tox.ini#L2, this is explicitly tested with PyPy.

File: pkgs/development/python-modules/yubico-client/default.nix [hunk 7]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, buildPythonPackage, isPyPy, fetchPypi
```
Comment (worldofpeace) [inline pkgs/development/python-modules/yubico-client/default.nix]:
```suggestion
{ stdenv, lib, buildPythonPackage, fetchPypi
```

File: pkgs/development/python-modules/yubico-client/default.nix [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+    sha256 = "0skkmrpvpb1pwyqjf3lh9vq46xagvwdx9kagpdbba2v5dgrk34d1";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  # pypi package missing test_utils and github releases is behind
+  doCheck = false;
+
+  meta = with lib; {
+    description = "verifying Yubico OTPs based on the validation protocol version 2.0";
```
Comment (worldofpeace) [inline pkgs/development/python-modules/yubico-client/default.nix]:
```suggestion
    description = "Verifying Yubico OTPs based on the validation protocol version 2.0";
```

**Pull request merged**